scripts/prodDeploy.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ *
+ * Deploys the factory and router contracts
+ *
+ * to run: 
+ * (testing on mumbai): npx hardhat run --network mumbai scripts/prodDeploy.ts
+ * (prod on polygon main): npx hardhat run --network polygon scripts/prodDeploy.ts
+ *
+ */
+
+import hre from "hardhat";
+
+const superfluidHost = "0x3E14dC1b13c488a8d5D310918780c983bD5982E7";
+const feeToSetter = "0xEBEddF028290cb0562f395699Ab764fd278a04cd";
+
+// delays x number of seconds
+const delay = (ms: number) => new Promise((res) => setTimeout(res, ms));
+
+const main = async () => {
+
+    // deploy factory
+    const Factory = await hre.ethers.getContractFactory("AqueductV1Factory");
+    const factory = await Factory.deploy(feeToSetter, superfluidHost);
+
+    console.log('factory: ', factory.address);
+
+    await delay(30000); // wait 30s
```

Comment:
Nitpick: Doesn't need to be updated but fyi `deploy` sends the deployment transaction but doesn't wait for the actual deployment. `await factory.deployTransaction.wait();` should await the receipt and negate the need for the `delay` here

https://docs.ethers.org/v5/api/contract/example/